conda/environment.yml

```diff
@@ -16,3 +16,5 @@ dependencies:
   - mamba
   - pip
   - sphinxcontrib-bibtex
+  - pip:
+    - jinja_markdown
```

Comment:
Suggest we merge #48 first, then update the conda lock files for this PR as well and get back on the train of passing builds/actions for the `main` branch

Reply:
unfortunately conda and pip still do not work well together... I think the `conda-lock lock` commands currently just completely ignore what comes after `pip:` There are some options, including 1. modify setupconda.yml to have an extra step to `pip install requirements.txt` (https://github.com/conda-incubator/conda-lock/issues/4)

Reply:
@scottyhq any idea when conda-lock 0.13.3 will be out? Should we wait for it or go with your extra step suggested above?

Reply:
no idea! so in order to keep things I'm in favor of adding an additional install `pip install jinja-markdown==1.210911`, which will need to happen *both* in setupconda.yml and for local installs after using the conda.lock files.

Reply:
My main concern with a manual pip install is resolving the dependencies of the library.
https://github.com/jpsca/jinja-markdown/blob/c96891722202aa1c88e4e6717ee2fa1de7e44ad3/setup.cfg#L19
Should we try to add those to the `environment.yml` and then do a:
```shell
pip install --no-deps jinja-markdown==1.210911
```
Also log an issue that documents this temporary solution until `conda-lock` is updated?

Reply:
see #61